src/java.base/share/classes/java/util/concurrent/Executors.java

```diff
@@ -524,30 +492,21 @@ public static <T> Callable<T> privilegedCallable(Callable<T> callable) {
 
     /**
      * Returns a {@link Callable} object that will, when called,
-     * execute the given {@code callable} under the current access
-     * control context, with the current context class loader as the
-     * context class loader. This method should normally be invoked
```

Comment:
We no longer state that the context class loader will be set when the callable is called, though the returned Callable _will_ still set the ccl. Is this OK?

Reply:
That's a good observation. Although deprecated for removal, the wording should at least specify what it does for the period before it is removed. I'll come up with some spec wording for this.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/0feceaae472710aa13e4c569f4d72d6149b7de07